UPGRADE-4.0.md

```diff
@@ -236,6 +236,9 @@ DependencyInjection
 
  * The `ExtensionCompilerPass` has been moved to before-optimization passes with priority -1000.
 
+ * Parameter `container.dumper.inline_class_loader` is configured by default with
```

Comment:
actually, the default is `false` also
it is `true` when starting a new Flex or Std edition though (in 3.4 or 4.0)